Reaktoro/Thermodynamics/Core/ChemicalEditor.cpp

```diff
@@ -412,14 +468,17 @@ struct ChemicalEditor::Impl
     auto createChemicalSystem() const -> ChemicalSystem
     {
         std::vector<Phase> phases;
-        phases.reserve(2 + mineral_phases.size());
+        phases.reserve(3 + mineral_phases.size());

```

Comment:
```suggestion
        const auto number_of_fluid_phases = 3;
        phases.reserve(number_of_fluid_phases + mineral_phases.size());
```
naming a magic number